docs/en/operation/StressBench.md

```diff
@@ -0,0 +1,77 @@
+---
+layout: global
+title: StressBench Framework
+nickname: StressBench
+group: Operations
+priority: 7
+---
+
+* Table of Contents
+  {:toc}
+
+The Alluxio system runs in different environments with different hardware for various purposes. The
+StressBench test suite is a performance benchmark suite to help you understand the performance of
+Alluxio in different scenarios. It will help you quickly test and analyze the Alluxio usage in your
+environment by simply running the command line.
+
+The StressBench is consisted of different tests to help you test the metadata and IO performance of
+the Alluxio system. The following benchmark tools are currently supported:
+* `StressMasterBench` - A benchmark tool measuring the master performance of Alluxio.
+* `StressClientBench` - A benchmark tool measuring the IO performance of Alluxio through the client.
+* `StressWorkerBench` - A benchmark tool measuring the IO performance of Alluxio with single node.
+* `UfsIOBench` - A benchmark tool measuring the IO performance to UFS.
+* `FuseIOBench` - A benchmark tool measuring the IO performance of Alluxio through Fuse interface.
+
```

Comment:
@jiacheliu3 is there any one-line summary for `RPCBench`?

Reply:
Actually `RpcBench` is a parent class with 3 children implementations. See if the lines below makes sense

```
RpcBench - A benchmark tool which simulates RPCs with a specified load and concurrency. Below are currently 3 implementations of RpcBench, each simulating one specific RPC. This tool is used to incur RPC stress on the master in a controlled manner.
RegisterWorkerBench - An implementation of RpcBench which simulates the RegisterWorker RPC.
WorkerHeartbeatBench - An implementation of RpcBench which simulates the WorkerHeartbeat RPC.
GetPinnedFileIdsBench - An implementation of RpcBench which simulates the GetPinnedFileIds RPC.
```

Reply:
I put the summary of `RpcBench` in this section, we can put the details of RPCBench in RPCBench section in following pr